webpack/webpack.common.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
       assert: require.resolve('assert'),
       buffer: require.resolve('buffer'),
       crypto: require.resolve('crypto-browserify'),
+      process: require.resolve('process'),
```

Comment:
webpack can find it automaticallly because it has the same name, but I find it better to mention it explicitely, so that we know more easily that it's actually being used.